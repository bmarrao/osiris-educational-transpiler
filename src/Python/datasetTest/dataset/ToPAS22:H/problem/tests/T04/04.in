5
4 1
A B C D
A B
4 1
A B C D
B C
4 1
A B C D
B A
4 1
A B C D
C B
4 1
A B C D
C A
