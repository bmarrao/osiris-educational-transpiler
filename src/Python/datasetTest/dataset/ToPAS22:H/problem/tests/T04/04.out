Possivel: 16
Possivel: 16
Possivel: 16
Possivel: 16
Possivel: 16
